signin.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>회원가입</title>
+    <link rel="stylesheet" href="./src/sign.css" />
+</head>
+<body>
+    <header>
+        <a class="header-logo" href="/index.html">
+            <img src="./images/logo.png" alt="홈으로 연결된 Linkbrary 로고" />
+        </a>
+        <p class="header-message">
+            회원이 아니신가요?<a class="header-link" href="signup.html">회원 가입하기</a>
+        </p>
+    </header>
+    <div class="sign-box">
+        <form class="sign-form">
+          <div class="sign-inputs">
+            <div class="sign-input-box">
+              <label class="sign-input-label">이메일</label>
+              <input class="sign-input" />
```

Comment:
type을 [email](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email)로 지정하면 email에 맞게 모바일 키패드 레이아웃이 변하고 email에 해당하는 입력값을 검증하도록 기본 동작하게 되어 있어요.

```suggestion
              <input type="email" class="sign-input" />
```